libraries/core/src/main/java/com/paritytrading/philadelphia/FIXValue.java

```diff
@@ -91,6 +91,74 @@ public int length() {
         return length;
     }
 
+    /**
+     * Compare the string representation of this value to the specified
+     * character.
+     *
+     * @param x the character to compare the string representation of this
+     *   value against
+     * @return true if this value represents the same string as the specified
+     *   character, otherwise false
+     */
+    public boolean contentEquals(byte x) {
+        return length == 1 && bytes[offset] == x;
```

Comment:
Why does this use `offset` while others don't?

Reply:
Good catch! 🔍 They should all use `offset`.

Reply:
I fixed these, and I also improved the test cases so that they exercise situations in which `offset != 0`.